### ISSUE TITLE
List AlphaWallet app on Foss Droid

### ISSUE THREAD

**Issue by colourfreak** [AlphaWallet/alpha-wallet-android#2229]
https://fossdroid.com/

**Comment by colourfreak** [comment]
@hboon Shall we transfer this issue to AlphaWallet-web3-provider repo?

**Comment by hboon** [comment]
@colourfreak no, it's good here. https://github.com/AlphaWallet/AlphaWallet-web3-provider is for the code that we inject into the dapp browser to support dapps.

**Comment by seabornlee** [comment]
OK, I am pushing.

**Comment by hboon** [comment]
Out of phase 1
